doc/introduction.tex

```diff
@@ -14,9 +14,12 @@
 
 \beginexample
 gap> LoadPackage("rig");
---
-RiG, A GAP package for racks, Version 0.9
-http://code.google.com/p/rig/
+-------------------------------------------------------------------
+Loading  rig 1 (A GAP package for racks, quandles and crossed sets)
```

Comment:
Yes, this looks odd, but it can be easily be improved: change the PackageName to `RiG`, and use a proper version, not just 1; e.g. "1.0"